src/main/java/uk/gov/gchq/palisade/data/serialise/AvroSerialiser.java

```diff
@@ -86,6 +86,8 @@ public void serialise(final Stream<O> objects, final OutputStream output) throws
                     dataFileWriter.append(next);
                 }
 
+            } catch (Exception ex) {
+                LOGGER.error("Error occurred: {}", ex.getMessage());
```

Comment:
Please rethrow the exception to ensure that it is propagated and handled :
`
```suggestion
                LOGGER.error("Error occurred: {}", ex.getMessage());
                throw new RuntimeException(ex)
```

Reply:
The rethrow has been added.